docs/dev/dev_environment.md

```diff
@@ -482,3 +482,17 @@ invoke generate-app-config-schema
 ```
 
 This command can only guess the schema, so it's up to the developer to manually update the schema as needed.
+
+### Test Data Generation
+
+To quickly generate test data for developing against this app, you can use the following command:
+
+!!! danger
```

Comment:
Can we move the danger below the bash command or above the "To quickly generate...".